pkg/attester/attester_test.go

```diff
@@ -0,0 +1,113 @@
+package attester
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	discovery "github.com/grafeas/grafeas/proto/v1beta1/discovery_go_proto"
+	grafeas "github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClient_Attest(t *testing.T) {
+	assert := assert.New(t)
+	ctx := context.Background()
+
+	name := "testAttester"
+	att := createAttester(name)
+
+	attestRequest := &AttestRequest{
+		ResourceURI: name,
+		Occurrences: []*grafeas.Occurrence{
+			{
+				Resource: &grafeas.Resource{
+					Uri: name,
+				},
+				NoteName: fmt.Sprintf("projects/rode/notes/%s", name),
+				Details: &grafeas.Occurrence_Discovered{
+					Discovered: &discovery.Details{
+						Discovered: &discovery.Discovered{
+							AnalysisStatus: discovery.Discovered_FINISHED_SUCCESS,
+						},
+					},
+				},
+			},
+		},
+	}
+
+	res, err := att.Attest(ctx, attestRequest)
```

Comment:
Can we make some assertions against the response here?  otherwise this test is only verifying that the call to `Attest` isn't failing.